Source/PyromaniacIsFun/NeedPyromania.cs

```diff
@@ -131,27 +131,27 @@ public float GainFromObservedFireInterval()
             {
                 var gainFromWildFire = numWildFire * Patcher.Settings.NeedPyromaniaGainPerWildFirePerDay;
                 gain += gainFromWildFire;
-                sb.AppendFormat("Saw wild fire x{0}: +{1:F0}% ~{2}\n",
-                    numWildFire, gainFromWildFire * 100,
-                    (PyromaniacUtility.ObservedWildFireDef.DurationTicks - minAgeWildFire).ToStringTicksToPeriod()
+                sb.AppendFormat("CF_PyromaniacIsFun_NeedPyrimania.SawWildFire".Translate(numWildFire, (gainFromWildFire * 100).ToString("F0"),
+                    (PyromaniacUtility.ObservedWildFireDef.DurationTicks - minAgeWildFire).ToStringTicksToPeriod())
+                    
```

Comment:
Suggest `sb.AppendLine` and remove `\n` from the string

Reply:
Applied